test/Test/Time/TypeSpec.hs

```diff
@@ -116,20 +116,20 @@ typeSpec_UnitNames ::
 
     "Lower"
     ~~~~~~~~~~~~
-         It "UnitName SecondUnit      = 's'"   (UnitName SecondUnit      `Is` "s")
-     -*- It "UnitName MillisecondUnit = 'ms'"  (UnitName MillisecondUnit `Is` "ms")
-     -*- It "UnitName MicrosecondUnit = 'mcs'" (UnitName MicrosecondUnit `Is` "mcs")
-     -*- It "UnitName NanosecondUnit  = 'ns'"  (UnitName NanosecondUnit  `Is` "ns")
-     -*- It "UnitName PicosecondUnit  = 'ps'"  (UnitName PicosecondUnit  `Is` "ps")
+         It "UnitName SecondUnit      = 's'"   (UnitName Second      `Is` "s")
```

Comment:
Probably in description it also worth to say `Second` instead of `SecondUnit` etc.